mobile/src/main/res/values-pl/strings.xml

```diff
@@ -78,27 +84,30 @@
     <string name="pref_second_notification_summary">Pokaż drugie powiadomienie bez treści, aby widoczna była dodatkowa ikona na pasku stanu.</string>
     <string name="pref_second_notification_icon">Druga ikona w pasku stanu</string>
     <string name="pref_second_notification_icon_summary">Styl małej ikony w pasku stanu w drugim powiadomieniu.</string>
-
+    <string name="pref_floating_widget">Pływający widget</string>
+    <string name="pref_floating_widget_summary">Pokazuje poziom glukozy, trend, deltę i czas jako pływający widget.\nNależy również aktywować powiadomienie, aby zapobiec zamknięciu aplikacji przez system Android.</string>
+    <string name="pref_floating_widget_size">Rozmiar</string>
+    <string name="pref_floating_widget_size_summary">Zmiana rozmiaru pływającego widżetu</string>
+    <string name="pref_floating_widget_style">Styl</string>
+    <string name="pref_floating_widget_style_summary">Styl pływającego widżetu.</string>
     <string name="pref_status_bar_icon_app">Ikona aplikacji</string>
     <string name="pref_status_bar_icon_glucose">Wartość glukozy</string>
     <string name="pref_status_bar_icon_trend">Strzałka trendu</string>
     <string name="pref_status_bar_icon_delta">Delta</string>
-
+    <string name="pref_wear_notification">Zegarek: wibracje</string>
     <string name="pref_global_broadcast">Transmisja globalna</string>
-
     <string name="select_receiver_show_all">Pokaż wszystko</string>
     <string name="select_receiver_no_glucodata_receiver">Nie znaleziono odbiornika!</string>
     <string name="button_ok">OK</string>
     <string name="button_cancel">Anuluj</string>
-
     <string name="menu_settings">Ustawienia</string>
-    <string name="menu_group_github">GitHub</string>
+    <string name="menu_group_github" translatable="false">GitHub</string>
     <string name="menu_update">Aktualizacje</string>
-    <string name="menu_support">Wsparcie</string>
+    <string name="menu_support" translatable="false">Wsparcie</string>
     <string name="menu_help">Pomoc</string>
-
-    <string name="help_link">https://github.com/pachi81/GlucoDataHandler/blob/master/README_PL.md</string>
-
+    <string name="help_link">https://github.com/pachi81/GlucoDataHandler/blob/master/README.md</string>
```

Comment:
Undo these lines -> README_PL.md and the lines with translatable="false" should not be part of this file

Reply:
Done